integration/oasis/stationgraph/graph_test.go

```diff
@@ -644,191 +644,191 @@ func TestGraphGeneral(t *testing.T) {
-							arrivalEnergy: 16,
-							chargeEnergy:  0,
-						},
-					},
-					&node{
-						id: 1,
-						neighbors: []*neighbor{
-							&neighbor{
-								targetNodeID: 3,
-								distance:     20,
-								duration:     20,
-							},
-							&neighbor{
-								targetNodeID: 4,
-								distance:     15,
-								duration:     15,
-							},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 0,
-							chargeEnergy:  16,
-						},
-					},
-					&node{
-						id: 2,
-						neighbors: []*neighbor{
-							&neighbor{
-								targetNodeID: 4,
-								distance:     5,
-								duration:     5,
-							},
-							&neighbor{
-								targetNodeID: 3,
-								distance:     30,
-								duration:     30,
-							},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 0,
-							chargeEnergy:  16,
-						},
-					},
-					&node{
-						id: 3,
-						neighbors: []*neighbor{
-							&neighbor{
-								targetNodeID: 5,
-								distance:     10,
-								duration:     10,
-							},
-							&neighbor{
-								targetNodeID: 6,
-								distance:     10,
-								duration:     10,
-							},
-							&neighbor{
-								targetNodeID: 7,
-								distance:     10,
-								duration:     10,
-							},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 0,
-							chargeEnergy:  16,
-						},
-					},
-					&node{
-						id: 4,
-						neighbors: []*neighbor{
-							&neighbor{
-								targetNodeID: 5,
-								distance:     15,
-								duration:     15,
-							},
-							&neighbor{
-								targetNodeID: 6,
-								distance:     15,
-								duration:     15,
-							},
-							&neighbor{
-								targetNodeID: 7,
-								distance:     15,
-								duration:     15,
-							},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 0,
-							chargeEnergy:  16,
-						},
-					},
-					&node{
-						id: 5,
-						neighbors: []*neighbor{
-							&neighbor{
-								targetNodeID: 8,
-								distance:     10,
-								duration:     10,
-							},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 999,
-							chargeEnergy:  999,
-						},
-					},
-					&node{
-						id: 6,
-						neighbors: []*neighbor{
-							&neighbor{
-								targetNodeID: 8,
-								distance:     20,
-								duration:     20,
-							},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 0,
-							chargeEnergy:  16,
-						},
-					},
-					&node{
-						id: 7,
-						neighbors: []*neighbor{
-							&neighbor{
-								targetNodeID: 8,
-								distance:     30,
-								duration:     30,
-							},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 0,
-							chargeEnergy:  16,
-						},
-					},
-					&node{
-						id: 8,
-						neighbors: []*neighbor{
-							&neighbor{},
-						},
-						chargeInfo: chargeInfo{
-							arrivalEnergy: 0,
-							chargeEnergy:  0,
-						},
-					},
-				},
-				startNodeID: 0,
-				endNodeID:   8,
-			},
-			[]nodeID{1, 4, 5},
-		},
+		// {
```

Comment:
This test will be re-enabled when merge with other pull, for full context please go to: https://github.com/Telenav/osrm-backend/pull/203